### ISSUE TITLE
gradle build failed

### ISSUE THREAD

**Issue by nyjsl** [nickbutcher/plaid#26]
//def gitCommitCount = Integer.parseInt('git rev-list --count HEAD'.execute([],project.rootDir).text.trim())
 'git rev-list --count HEAD'.execute([],project.rootDir).text.trim()  can not be cast into Integer
 i just wrote def gitCommitCount = n instead ..

**Comment by bbjsmart** [comment]
Error:(33, 0) Could not find property 'gitCommitCount' on ProductFlavor_Decorated{name=main, minSdkVersion=ApiVersionImpl{mApiLevel=21, mCodename='null'}, targetSdkVersion=ApiVersionImpl{mApiLevel=23, mCodename='null'}, renderscriptTargetApi=null, renderscriptSupportModeEnabled=null, renderscriptNdkModeEnabled=null, versionCode=null, versionName=null, applicationId=io.plaidapp, testApplicationId=null, testInstrumentationRunner=null, testHandleProfiling=null, testFunctionalTest=null, signingConfig=null, resConfig=null, mBuildConfigFields={}, mResValues={}, mProguardFiles=[], mConsumerProguardFiles=[], mManifestPlaceholders={}}.
<a href="openFile:D:\code\demo\plaid-master\plaid-master\app\build.gradle">Open File</a>

**Comment by bbjsmart** [comment]
Error:(20, 0) CreateProcess error=2, ϵͳÕҲ»µ½ָ¶
<a href="openFile:D:\code\demo\plaid-master\plaid-master\app\build.gradle">Open File</a>

**Comment by bu2zhouzhu** [comment]
I have the same problem.

**Comment by wang-qian** [comment]
me too

**Comment by bu2zhouzhu** [comment]
OK. I get rid of it now. Here is my way:
[Download](http://www.git-scm.com/download/win) and install *git for window*. There is a choice in installation wizard you should **not** use the default *Bash Only*, but choose *Bash and Windows Prompt*. After the installation finished, edit the path environment variable, in the end add " ;C:\Program Files\Git\bin  ". (By choosing *Bash and Windows Prompt*, the installation program will add " C:\Program Files\Git\cmd " to your Path environment varialbe. But on my PC it's not enough. I need also add Git\bin). Then restart your Android Studio. The problem should be gone.

**Comment by BillBosiolis** [comment]
I get the same error

**Comment by nickbutcher** [comment]
Dupe of #17. Building requires git on your path.

**Comment by BillBosiolis** [comment]
On Ubuntu, with Git installed (and being on the path) I had no problem to open the project in Android Studio and build it successfully.

On Windows, I was getting the known error (20 - CreateProcess error=2). Here's what I did to solve it

1. Installed a newer Git version (I don't think it has to do anything with the version, I just wanted to modify the installation)
2. In the "Adjusting your Path environment" step I selected the second option - Run Git from the Windows Command Prompt
3. Restarted Studio and set the the path to the git executable. Settings -> Version Control -> Git and set it "C:\Program Files\Git\cmd\git.exe".

Hope that it helps
